addons/account/wizard/account_invoice_send.py

```diff
@@ -23,6 +23,11 @@ class AccountInvoiceSend(models.TransientModel):
         domain="[('model', '=', 'account.move')]"
         )
 
+    # View fields
+    move_types = fields.Char('Move types', store=False, readonly=True,
+                             help='Technical field containing a textual representation of the selected move types,'
+                                  'if multiple. It is used to inform the user in the window in such case.')
```

Comment:
Be careful with your multi-line strings like this. There was missing a space between the comma of the first line and the first word of the second line.

I don't think it hurts to store the field on a `TransientModel`, and it reduces the risk of bugs. However, computing with a compute function it (and thus not storing) would be better. It will always work, not only when coming from  `action_send_and_print`

I like to either inline everything, or really make it on multiple lines.

```suggestion
    move_types = fields.Char(
        string='Move types', 
        store=False, 
        readonly=True,
        help='Technical field containing a textual representation of the selected move '
             'types, if multiple. It is used to inform the user in the window in such case.',
    )
```

Reply:
Alright, I'll do it like that! Thank you for the feedback!